### DISCUSSION TITLE
Skip confirmation if there is only 1 pane open when quitting Wezterm

### DISCUSSION THREAD

**Discussion by pengux** [wezterm/wezterm#2654] [Q&A] [answered]
If there is only 1 pane open when quitting Wezterm, I would like to skip confirmation and just close Wezterm. Is this possible?

**Comment by wez** [comment, marked as answer]
The following options affect closing:
* https://wezfurlong.org/wezterm/config/lua/config/exit_behavior.html
* https://wezfurlong.org/wezterm/config/lua/config/clean_exit_codes.html
* https://wezfurlong.org/wezterm/config/lua/config/skip_close_confirmation_for_processes_named.html
* https://wezfurlong.org/wezterm/config/lua/mux-events/mux-is-process-stateful.html

That last one is a hook; you might consider using the [mux](https://wezfurlong.org/wezterm/config/lua/wezterm.mux/index.html) module to count up all of the windows/tabs/panes and use that to influence the return value.

**Comment by pengux** [comment]
Thank you, https://wezfurlong.org/wezterm/config/lua/config/skip_close_confirmation_for_processes_named.html actually fits my use case so I used that instead.
